packages/ui/src/accounts/modals/MoveFoundsModal/MoveFundsModalButtons.tsx

```diff
@@ -13,22 +13,29 @@ interface MoveFundsModalItemProps {
   noFreeAccounts: boolean
 }
 
+interface LinkButtonProps {
+  to: string
+  label: string
+  onClick: () => void
+  hide?: boolean
+}
+
+const LinkButton = ({ to, label, onClick, hide = false }: LinkButtonProps) => (
+  <div onClick={onClick}>
+    <LinkButtonPrimary size="medium" to={to}>
+      {label}
+    </LinkButtonPrimary>
+  </div>
+)
```

Comment:
I don't see where is the `hide` used. Am I missing something ?

IMO just deleting the button or commenting it would be fine.

Reply:
ab3ad24 finishes the intended implementation. Ready to remove though.